.circleci/config.yml

```diff
@@ -210,6 +226,13 @@ jobs:
 
 workflows:
   version: 2
+  lint:
+    jobs:
+      - setup_github_package_registry:
+          context: github-package-registry
```

Comment:
You probably don't need this anymore since the repo is open, though not certain about that.

Reply:
[Nope](https://circleci.com/gh/keep-network/keep-ecdsa/5969?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link),  `Unable to authenticate, need: Basic realm="GitHub Package Registry"`  😞

Reply:
I think this happened for a couple of reasons:
- You need to update `package-lock` by running `npm i`.
- You need to drop `.npmrc`.
- You need to drop the `publishConfig`.

However, I'll do all of this stuff in one step when I handle deployment later today.